tests/e2e/performance/test_pod_reattachtime.py

```diff
@@ -76,13 +76,13 @@ def test_pvc_reattach_time_performance(self, pvc_factory, teardown_factory):
         # Create a pod on one node
         logging.info(f"Creating Pod with pvc {pvc_obj.name} on node {node_one}")
 
-        helpers.pull_images("nginx")
+        helpers.pull_images("quay.io/ocsci/perf:latest")
```

Comment:
should this path be added to the constant file?

Reply:
will add it as constant.

Reply:
done